### ISSUE TITLE
Sigmoid function applied twice in Discriminator's output. 

### ISSUE THREAD

**Issue by Kadam-Tushar** [qianlima-lab/AJ-RNN#1]
[Here](https://github.com/dnldsht/AJ-RNN/blob/next/ajrnn.py#L169) in the final layer of the discriminator `sigmoid` function is applied and then [here](https://github.com/dnldsht/AJ-RNN/blob/b011ea1c2c9ca8674f45a0741240a7adc7274583/ajrnn.py#L215) while calculating loss, outputs are considered as logits hence TensorFlow will apply sigmoid over those outputs of Discriminator layer.  I referred to documentation to confirm this. [sigmoid_cross_entropy_with_logits](https://www.tensorflow.org/api_docs/python/tf/nn/sigmoid_cross_entropy_with_logits)
